cuvs-bench/README.md

```diff
@@ -0,0 +1,90 @@
+# cuVS Benchmarks Docker Images
+
+This folder contains the Dockerfiles for generating GPU and CPU cuVS benchmark images.
+
+These images are meant to enable end users of cuVS ANN algorithms to easily run and reproduce benchmarks and comparisons between cuVS and third party libraries.
+
+# Image types:
+
+There are two image types:
+
+- gpu: Contains dockerfile to build images using conda packages for GPU systems.
+- cpu: Contains dockerfile to build images using conda packages for CPU systems. Based on `mambaforge`.
+
+# Running the Containers
+
+For complete details, refer to the cuVS documentation: https://docs.rapids.ai/api/cuvs/nightly/cuvs_bench/#installing-the-benchmarks
+
+We provide images for GPU enabled systems, as well as systems without a GPU. The following images are available:
+
+- `cuvs-bench`: Contains GPU and CPU benchmarks, can run all algorithms supported. Will download million-scale datasets as required. Best suited for users that prefer a smaller container size for GPU based systems. Requires the NVIDIA Container Toolkit to run GPU algorithms, can run CPU algorithms without it.
+- `cuvs-bench-datasets`: Contains the GPU and CPU benchmarks with million-scale datasets already included in the container. Best suited for users that want to run multiple million scale datasets already included in the image.
+- `cuvs-bench-cpu`: Contains only CPU benchmarks with minimal size. Best suited for users that want the smallest containers to reproduce benchmarks on systems without a GPU.
+
+Nightly images are located in [DockerHub](https://hub.docker.com/r/rapidsai/cuvs-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/cuvs-bench).
```

Comment:
@dantegd You marked my previous conversation as resolved but didn't answer the question -- what do we do with NGC here? This link is pointing to DockerHub. I observe a similar issue in https://github.com/rapidsai/cuvs/blob/f708fe388ee206105fd7894388283995e88ab7f9/docs/source/cuvs_bench/index.rst?plain=1#L96

Reply:
Created an issue so that we add the link when the container is published https://github.com/rapidsai/docker/issues/716, I'm not entirely sure how the link will look like.